### ISSUE TITLE
Kendo Grid with batch editing doesn't update the viewmodel

### ISSUE THREAD

**Issue by aleksanderson** [kendo-labs/knockout-kendo#54]
Hi, 

I don't  know what is wrong with my implementation but I can't get my viewmodel updated if I'm using editable KendoUI Grid with Knockout-Kendo. If I change some particular table field and log the viewmodel it will not get any update.
You can see an example here: http://jsfiddle.net/aleksanderson/hd5c8/

What is the reason of such behavior?

Thanks in advance.

**Comment by kiamoyman** [comment]
@luferogo, were you able to figure out how to deal with CRUD operations?

**Comment by luferogo** [comment]
@ronniemacapobre ,  Nope. We just decided to use the grid's "transport" methods , and in the read method pass the ko.toJS. So we do not have a real bounded grid to ko model.
